### PR TITLE
Resolve calculator problems

### PR DESCRIPTION
**Pull request by gabrielgiroe1**

### CODE REVIEW

File: 03_simon_says/simon_says.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def   repeat(var)
+  if
+end
```
Comment (pirvudoru) [inline 03_simon_says/simon_says.rb]: I think this needs to be on a separate PR. please remove the file from this branch :)

File: 02_calculator/calculator.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+def subtract(a,b)
+  return (a-b)
+end
+
+def sum (arr)
+  sum=0.0
+  if arr.length<=0 then
+    return  0
+  end
```
Comment (pirvudoru) [inline 02_calculator/calculator.rb:13]: Why do you need this `if` here. Does it matter if you remove it or keep it for the output of the function ? (aka do tests continue to pass?)

-- try to minimize exceptional cases when you write code

File: 02_calculator/calculator.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+def add(a,b)
+  a+b
+end
+
+def subtract(a,b)
+  return (a-b)
```
Comment (pirvudoru) [inline 02_calculator/calculator.rb]: You don't need `( ...  )`
And you don't need `return` -- ruby returns by default the result of the last line of code in the function. Should be similar to what you did for the `add` function

File: 02_calculator/calculator.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  end
+
+  return sum
+end
+
+def multiply(base, exp)
+  var1=1
+  var=base
+
+  while var<=exp
```
Comment (pirvudoru) [inline 02_calculator/calculator.rb]: Does `base ** exp` have the same result?

File: 02_calculator/calculator.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+def sum (arr)
+  sum=0.0
+  if arr.length<=0 then
+    return  0
+  end
+
+  arr.each do |item |
+    sum+=item
+  end
```
Comment (pirvudoru) [inline 02_calculator/calculator.rb]: There's something simpler https://ruby-doc.org/core-3.0.1/Enumerable.html#method-i-sum

**Review by pirvudoru — CHANGES_REQUESTED** [review]

File: 02_calculator/calculator.rb (hunk 4 above)
Comment (gabrielgiroe1) [reply 02_calculator/calculator.rb]: yes, but I don't see this function when I wrote
